### DISCUSSION TITLE
Can't find cfx tokens on trust wallet

### DISCUSSION THREAD

**Discussion by Ahmedfawaz11** [community/community#147041] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I transferred cfx tokens to trust wallet and history show the process is done but tokens don't show in my balance..is there any help please?

**Comment by shinybrightstar** [comment]
Hi @Ahmedfawaz11 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
